pkgs/by-name/te/termbg/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "termbg";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "dalance";
+    repo = "termbg";
+    rev = "v${version}";
+    hash = "sha256-KLWfdA7TArJqYoxIXQavaTNw/lmWQN9aeltxssIUEvk=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
+
+  meta = {
+    description = "A Rust library for terminal background color detection";
```

```suggestion
    description = "Program for terminal background color detection";
```
This only packages the binary.